app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -9,7 +9,7 @@
 
       <ul>
         <li>teachers who make a sustained contribution to their school could expect to earn over £43k</li>
```

Comment:
Should this be the salarywithtlr variable?

Reply:
I've ticketed this up in a separate TLR ticket - I'll create a PR for this next week!